### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by vladzhorov**

### CODE REVIEW

**Review by moltenessence — APPROVED** [review]

File: ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                Price = 150.0m,
+                DurationInMinutes = 90,
+                Location = "New Location",
+                UserProfileId = userProfileId,
+                AssistanceCategoryId = assistanceCategoryId
+            };
+
+            // Act
+            var response = await _client.PostAsJsonAsync($"{ApiRoutes.Assistances}", createViewModel);
+            response.EnsureSuccessStatusCode();
```
Comment (Mmishaaa) [inline ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceControllerTest.cs:132]: same

File: ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceControllerTest.cs [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+        [Fact]
+        public async Task GetById_ReturnsAssistanceViewModel()
+        {
+            // Arrange
+            var assistanceId = _assistanceId;
+
+            // Act
+            var response = await _client.GetAsync($"{ApiRoutes.Assistances}/{assistanceId}");
+            response.EnsureSuccessStatusCode();
```
Comment (Mmishaaa) [inline ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceControllerTest.cs:107]: same

File: ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceControllerTest.cs [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        [Fact]
+        public async Task GetAll_ReturnsListOfAssistanceViewModels()
+        {
+            // Arrange
+            var pageNumber = 1;
+            var pageSize = 10;
+
+            // Act
+            var response = await _client.GetAsync($"{ApiRoutes.Assistances}?pageNumber={pageNumber}&pageSize={pageSize}");
+            response.EnsureSuccessStatusCode();
```
Comment (Mmishaaa) [inline ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceControllerTest.cs:91]: same

File: ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceCategoryControllerTest.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+        [Fact]
+        public async Task Delete_ShouldDeleteAssistanceCategory()
+        {
+            // Arrange
+            var category = _testHelper.GetFirstCategory();
+
+            // Act
+            var response = await _client.DeleteAsync($"{ApiRoutes.AssistanceCategories}/{category.Id}");
+            response.EnsureSuccessStatusCode();
```
Comment (Mmishaaa) [inline ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceCategoryControllerTest.cs:96]: same

File: ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceCategoryControllerTest.cs [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        [Fact]
+        public async Task Update_ShouldUpdateAssistanceCategory()
+        {
+            // Arrange
+            var category = _testHelper.GetFirstCategory();
+            var updateViewModel = AssistanceCategoryTestHelper.UpdateAssistanceCategoryViewModel();
+
+            // Act
+            var response = await _client.PutAsJsonAsync($"{ApiRoutes.AssistanceCategories}/{category.Id}", updateViewModel);
+            response.EnsureSuccessStatusCode();
```
Comment (Mmishaaa) [inline ServiceFinder/ServiceFinder.IntegrationTest/IntegrationTest/AssistanceCategoryControllerTest.cs:80]: same

**Review by Mmishaaa — CHANGES_REQUESTED** [review]
and in other cases as well

**Review by Mmishaaa — APPROVED** [review]

**Pull request merged**